cesium/internal/unary/writer.go

```diff
@@ -181,10 +194,12 @@ func (w *Writer) len(dw *domain.Writer) int64 {
 
 // Write validates and writes the given array.
 func (w *Writer) Write(series telem.Series) (a telem.AlignmentPair, err error) {
+	defer func() { err = w.wrapError(err) }()
```

Comment:
Weird but I guess I'm ok with it

Reply:
I don't know how much value this defer statements adds though. Looks ugly to me. If you want to keep I'm fine with it though